packages/grant-explorer/src/features/round/ViewRoundPage.tsx

```diff
@@ -869,7 +868,7 @@ const RoundStatsTabContent = ({
   roundId: string;
   round: Round;
   chainId: ChainId;
-  token?: VotingToken;
+  token?: PayoutToken;
```

Comment:
why did the type change here?

Reply:
The types are different and we should consider adding GTC to votingToken. Maybe you can change it back and raise an additional PR?

Reply:
done! https://github.com/gitcoinco/grants-stack/pull/3418/commits/3e331afff63a7ae856c6785dacfb391362dc33cf

here's the additional PR: https://github.com/gitcoinco/grants-stack/pull/3425